### PR TITLE
Fix Vectara filter

### PR DESCRIPTION
**Pull request by 3eif**
This PR fixes the Vectara metadata filter type

### CODE REVIEW

File: packages/components/nodes/vectorstores/Vectara_Existing/Vectara_Existing.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ class VectaraExisting_VectorStores implements INode {
             {
                 label: 'Vectara Metadata Filter',
                 name: 'filter',
-                type: 'json',
+                type: 'string',
                 additionalParams: true,
```
Comment (HenryHengZJ) [inline packages/components/nodes/vectorstores/Vectara_Existing/Vectara_Existing.ts:38]: can we add a `description` with how to use the filter, or the link to the docs page? Something like:

```
Refer to <a target="_blank" href="https://docs.vectara.com">official guide</a> on how to use filter
```

Comment (3eif) [reply packages/components/nodes/vectorstores/Vectara_Existing/Vectara_Existing.ts:38]: This has been added in https://github.com/FlowiseAI/FlowiseDocs/pull/22

Comment (HenryHengZJ) [reply packages/components/nodes/vectorstores/Vectara_Existing/Vectara_Existing.ts:38]: thanks for the docs PR! would be good to add a description property as well

`description: 'Refer to....'`

Comment (3eif) [reply packages/components/nodes/vectorstores/Vectara_Existing/Vectara_Existing.ts:38]: Ah I didn't know there was a description property. Will add that as well!

Comment (3eif) [reply packages/components/nodes/vectorstores/Vectara_Existing/Vectara_Existing.ts:40]: @HenryHengZJ Added the description property. Let me know if there is anything else I should change.

**Review by HenryHengZJ — APPROVED** [review]
looks good! thanks @3eif

**Comment by Josephrp** [comment]
thank you for this ! 🚀

**Pull request merged**